src/content/decentralized-identity.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Decentralized identity 
+description: What is decentralized identity, and why does it matter?
+lang: en
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Centralized identity systems are fragmented, poorly secured, and offer minimal privacy protection.
```

Comment:
The rest of the page is looking really good... I think we could strengthen these summary points now based on the latest the page has to offer.

Would consider along the lines of
1. Traditionally identity systems have centralized the issuance, maintenance and control of your identifiers.
2. Decentralized identity solutions give the power back to the individual, further decreasing reliance on centralized third-parties
3. Thanks to crypto, users now have the tools to issue, hold and control their own identifiers and attestations once again

Reply:
Just added this now.